### PR TITLE
Config with zope product

### PR DESCRIPTION
**Pull request by icemac**
* Lint the code.
* Add support for Python 3.9.

### CODE REVIEW

**Comment by icemac** [comment]
Interestingly PyPy3 breaks locally with:

```python
Error in test testISO8601 (DateTime.tests.test_datetime.DateTimeTests)
Traceback (most recent call last):
  File "/opt/local/lib/pypy37/lib-python/3/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/opt/local/lib/pypy37/lib-python/3/unittest/case.py", line 628, in run
    testMethod()
  File ".../DateTime/src/DateTime/tests/test_datetime.py", line 413, in testISO8601
    isoDt = DateTime('20020502T080000-400')
  File ".../DateTime/src/DateTime/DateTime.py", line 447, in __init__
    return self._parse_args(*args, **kw)
  File ".../DateTime/src/DateTime/DateTime.py", line 758, in _parse_args
    yr, mo, dy, hr, mn, sc, tz = self._parse(arg, datefmt)
  File ".../DateTime/src/DateTime/DateTime.py", line 998, in _parse
    raise SyntaxError(st)
DateTime.interfaces.SyntaxError: 20020502T080000-400
```

File: setup.py [hunk 1]
```diff
@@ -23,7 +26,7 @@
 with open(os.path.join(here, 'CHANGES.rst')) as f:
     CHANGES = f.read()
 
-version = '5.0.dev0'
+version = '4.3.dev0'
```
Comment (dataflake) [inline setup.py:29]: This should be `4.4.dev0`

**Review by dataflake — APPROVED** [review]
I couldn't look at the PyPy issue, Homebrew's PyPy is broken right now so I have no working interpreter.

**Comment by icemac** [comment]
> I couldn't look at the PyPy issue, Homebrew's PyPy is broken right now so I have no working interpreter.

Let's rely on GHA than, it has no problems there.

File: setup.py (hunk 1 above)
Comment (icemac) [reply setup.py]: Good catch, fixed in https://github.com/zopefoundation/DateTime/pull/29/commits/94e62c3485cb4a5bf9915495a1525bb2086d9737

**Pull request merged**